App/Redux/PhotoViewingRedux.ts

```diff
@@ -3,14 +3,14 @@ import Config from 'react-native-config'
 
 import { IFiles, Thread } from '@textile/react-native-sdk'
 
-interface ThreadConfig {
+export interface ThreadConfig {
```

Comment:
yep, I can hear aaron's alarms for sure, given that this now needs to be exported instead of keeping the logic tight in this file.